nemo/collections/nlp/parts/mixins/nlp_adapter_mixins.py

```diff
@@ -227,8 +227,8 @@ def add_adapter(self, peft_cfgs: Union[PEFTConfig, List[PEFTConfig]]):
             peft_cfgs: One or more PEFTConfig objects that specify the PEFT method configuration
         """
 
-        if self.cfg.optim.name == "distributed_fused_adam":
-            raise ValueError('distributed_fused_adam is not supported for PEFT. Please use fused_adam')
+        # if self.cfg.optim.name == "distributed_fused_adam":
+        #     raise ValueError('distributed_fused_adam is not supported for PEFT. Please use fused_adam')
```

Comment:
@zhehuaichen should we remove this code or restore before merging?

Reply:
restore